.github/workflows/build-sw-container.yml

```diff
@@ -16,10 +16,21 @@ jobs:
 
     - name: Generate tag list
       id: generate-tag-list
+      env:
+        REPO: ${{ matrix.repo }}
+        TIMESTAMP: ${{ needs.make-date-tag.outputs.dtag }}
       run: |
         docker_repo=${GITHUB_REPOSITORY/opensciencegrid\/docker-/opensciencegrid/}
-        tag_list=$docker_repo:latest,$docker_repo:${GITHUB_REF##*/v}
-        echo "::set-output name=taglist::$tag_list"
+        tag_list=()
+        for registry in hub.opensciencegrid.org docker.io; do
+          for image_tag in "$REPO" "$REPO-$TIMESTAMP"; do
+            tag_list+=("$registry/$docker_repo":"$image_tag")
+          done
+        done
+        # This causes the tag_list array to be comma-separated below,
+        # which is required for build-push-action
+        IFS=,
+        echo "::set-output name=taglist::${tag_list[*]}"
```

Comment:
This list of tags is very different from what we had previously (`opensciencegrid/topology:latest` and `opensciencegrid/topology:X.Y.Z` where `X.Y.Z` is a version number)